src/main/java/domain/Rate.kt

```diff
@@ -23,7 +31,7 @@ class Rate : Serializable {
     constructor() {}
 
     constructor(vehicleType: VehicleType, kmPrice: Double, vignetteType: VignetteType) {
-        this.id = UUID.randomUUID().toString()
+        //this.id = UUID.randomUUID()
```

Comment:
Why is this commented? Either use it or remove it, we can get this back from git history

Reply:
Fixed